mongodbatlas/cloud_provider_access.go

```diff
@@ -52,7 +52,12 @@ type AWSIAMRole struct {
 	FeatureUsages              []*FeatureUsage `json:"featureUsages,omitempty"`              // Atlas features this AWS IAM role is linked to.
 	IAMAssumedRoleARN          string          `json:"iamAssumedRoleArn,omitempty"`          // ARN of the IAM Role that Atlas assumes when accessing resources in your AWS account.
 	ProviderName               string          `json:"providerName,omitempty"`               // Name of the cloud provider. Currently limited to AWS.
-	RoleID                     string          `json:"roleId,omitempty"`                     // Unique ID of this role.
+	RoleID                     string          `json:"roleId,omitempty"`                     // Unique 24-hexadecimal digit string that identifies the role.
+	ID                         *string         `json:"_id,omitempty"`                        // Unique 24-hexadecimal digit string that identifies the Azure Service Principal in Atlas.
+	AtlasAzureAppID            *string         `json:"atlasAzureAppId,omitempty"`            // Azure Active Directory Application ID of Atlas.
```

Comment:
It will be good to  change order of those to make it easier for people to understand what comes from which polymorphic type. For example: Putting ID on the top and adding Azure prefix in the comments

Reply:
Just to make sure I understood the suggestion here: you are proposing to update the description of each field and add `Azure:desciption` or `AWS: description`?  What about the fields that are needed with both?

Reply:
I have no strong opinion on this. 
My suggestion was to add prefixes to Azure: specific properties. Adding AWS ones could be a bonus. 
Otherwise no description update is needed.

Reply:
updated